app/perf_overlay.conf

```diff
@@ -1,3 +1,4 @@
 CONFIG_PERFORMANCE_COUNTERS=y
 CONFIG_SYS_HEAP_RUNTIME_STATS=y
-CONFIG_TIMING_FUNCTIONS=y
\ No newline at end of file
+CONFIG_TIMING_FUNCTIONS=y
+CONFIG_ADSP_IDLE_CLOCK_GATING=n
```

Comment:
@abonislawski I think a comment here would not hurt to explain use of xtensa timer and clock gating are not compatible. Even better would be to have this at Kconfig level but a comment here goes a long way.